makefile

```diff
@@ -35,6 +36,19 @@ release: | tmp/output/release/
 	@test $(VERSION)
 	gh release create $(VERSION) --notes "New release." --title "$(VERSION)" tmp/output/release/*
 
+xxx: tmp/input/Orphanet_Nomenclature_Pack_EN/ORPHA_ICD11_mapping_en_newversion_2023.xml
+
+# Mappings
+tmp/input/Orphanet_Nomenclature_Pack_EN/ORPHA_ICD11_mapping_en_newversion_2023.xml: | tmp/input/
```

Comment:
## Prerequisite dependency: will break in the future
I want to make this reproducible. For that, the "API" I'm interacting with needs to be stable, but in this case there is a file source issues.

**File name**
I need to use `ORPHA_ICD11_mapping_en_newversion_2023.xml`. However, there are two things that are unstable about this filename: (i) `_2023`, and (ii) `_newversion`. I imagine that in the future (i) will change according to the year / date, and (ii) will probably be dropped from the filename or renamed. I think this 'new version' may supercede the old version of the file eventually.

**Download URL**
Further, I don't have a URL for this release specifically; it doesn't exist yet. I only have the URL for the 'latest' release: https://www.orphadata.com/data/nomenclature/packs/Orphanet_Nomenclature_Pack_EN.zip. Whenever 2023 does get archived as a previous release, it should be accessible via this URL: https://www.orphadata.com/data/nomenclature/previous_versions/Orphanet_Nomenclature_Pack_EN_2023.zip

Reply:
Thank you for this analysis.. This is a bit of a pity, but in our case, we can use your pipeline for the 1 time alignment, and then work with Orphanet to provide a stable mapping.

Reply:
@matentzn Actually, it may be the case that it will be stable soon. Julie just sent me an email, and she said that the ICD11Foundation references will also appear in the alignment file, which has a stable URL and filename, starting July this year. I've set myself a reminder on my calendar to check.